lms/views/admin/application_instance.py

```diff
@@ -301,10 +303,13 @@ def update_instance(self):
         )
 
         for setting, sub_setting, setting_type in (
-            ("canvas", "sections_enabled", bool),
-            ("canvas", "groups_enabled", bool),
             ("blackboard", "files_enabled", bool),
             ("blackboard", "groups_enabled", bool),
+            ("canvas", "sections_enabled", bool),
+            ("canvas", "groups_enabled", bool),
+            ("desire2learn", "client_id", str),
+            ("desire2learn", "client_secret", AESService.AESSecret),
```

Comment:
I think we can define a local value here for this? Something like `AES_SECRET = object()`?